src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -0,0 +1,15 @@
+package traffic_simulation
+
+fun main(args: Array<String>) {
+
+    scenario(testList, testRoad)
```

Comment:
Übrigens, dass etwas eine Liste ist, ist meistens wesentlich weniger interessant als was in der Liste drin ist. Deswegen braucht mal `List` nicht in den Namen aufzunehmen - außerdem kann mir das ja auch die IDE sagen, wenn ich es wissen will.